pkgs/development/python-modules/djangorestframework-jsonp/default.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  django,
+  python,
+  pytest-django,
+  pytestCheckHook,
+  djangorestframework,
+}:
+
+buildPythonPackage rec {
+  pname = "djangorestframework-jsonp";
+  version = "1.0.2";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
please see the language docs for details